themes/theme-gmd/pages/Category/components/Content/index.jsx

```diff
@@ -1,7 +1,6 @@
 import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
-import { ResponsiveContainer } from '@shopgate/engage/components';
-import { GlobalLocationSwitcher } from '@shopgate/engage/locations';
+import ProductFilters from 'Components/ProductFilters';
```

Comment:
When running Engage in the "app mode", the filter bar overlaps the product list on desktop safari / iOS app.
![Bildschirmfoto 2020-08-13 um 14 06 20](https://user-images.githubusercontent.com/13087792/90135350-d2bf8b00-dd72-11ea-84cd-a14523693b9c.png)